[OSDOCS-7705] machine pool what's new (#64598)
openshift/openshift-docs: rosa_release_notes/rosa-release-notes.adoc

[id="rosa-q3-2023_{context}"]
=== Q3 2023

* **Shared virtual private clouds:** ROSA now supports installing clusters into VPCs shared among AWS accounts that are part of AWS organizations. AWS account installing ROSA cluster can now use shared subnets owned by a management account. For more information, see xref:../rosa_install_access_delete_clusters/rosa-shared-vpc-config.html[Configuring a shared virtual private cloud for ROSA clusters].

* **Machine pool disk volume size:** You can now configure your machine pool disk volume size for additional flexibility. You can select your own sizing for the disk volumes of their worker machine pool nodes. For more information, see xref:../rosa_cluster_admin/rosa_nodes/rosa-managing-worker-nodes.html#configuring_machine_pool_disk_volumerosa-managing-worker-nodes[Configuring machine pool disk volume].

> **Machine pool update:** Users can now add and delete taints during machine pool creation. This new feature provides more flexibility and cost-effectiveness for cluster administrators, specifically in regards to scaling infrastructure based on changing resource requirements. For more information, see xref:../rosa_cluster_admin/rosa_nodes/rosa-managing-worker-nodes.html#creating_a_machine_pool_rosa-managing-worker-nodes[Creating a machine pool].

Reviewer: ```suggestion
**Machine pool update:** Users can now add taints to and delete the machine pool that is automatically generated when a new cluster is created . This new feature provides more flexibility and cost-effectiveness for cluster administrators, specifically in regards to scaling infrastructure based on changing resource requirements. For more information, see xref:../rosa_cluster_admin/rosa_nodes/rosa-managing-worker-nodes.html#creating_a_machine_pool_rosa-managing-worker-nodes[Creating a machine pool].
```

> * **Machine pool update:** The machine pool that is automatically generated during cluster creation can now be deleted and can have taints added to it. This new feature provides more flexibility and cost-effectiveness for cluster administrators, specifically in regards to scaling infrastructure based on changing resource requirements. For more information, see xref:../rosa_cluster_admin/rosa_nodes/rosa-managing-worker-nodes.html#creating_a_machine_pool_rosa-managing-worker-nodes[Creating a machine pool].

Reviewer 2: ```suggestion
* **Machine pool update:** You can now delete the machine pool that is automatically generated during cluster creation. If you do not delete this machine pool, you can add taints to it. This new feature provides more flexibility and cost-effectiveness for cluster administrators, specifically in regards to scaling infrastructure based on changing resource requirements. For more information, see xref:../rosa_cluster_admin/rosa_nodes/rosa-managing-worker-nodes.html#creating_a_machine_pool_rosa-managing-worker-nodes[Creating a machine pool].
```
- I don't understand the first sentence. If you delete the machine pool, how can you still add taints to it? The way the sentence is phrased makes it sound like this is possible.
- Consider breaking up the first sentence to clarify the meaning and to remove the passive voice. I've made a suggestion above, but I'm not sure if this actually changes the meaning. Lmk if you want me to take another look. Thx!

Author: CC: @mletalie to take a look at the new draft.
